site/deno.jsonc

```
@@ -0,0 +1,39 @@
+{
+  "lock": false,
+  "nodeModulesDir": "auto",
+  "fmt": {
+    "proseWrap": "preserve"
+  },
+  "tasks": {
+    "setup": "deno task docs:index-versions && deno task docs:genapi",
```

Maybe we could combine the scripts so we can just run `deno task setup`?

```suggestion
    "setup": "deno install --allow-scripts && deno task docs:index-versions && deno task docs:genapi",
```

---

Ah yeah that is great